src/main/resources/flows-config.yaml

```diff
@@ -52,3 +52,5 @@ flow:
       - name: personalInfo
   personalInfo:
     nextScreens: null
+  sensitiveQuestionsWarning:
+    nextScreens: null
```

Comment:
@lkemperman-cfa since the previous pages haven't been built yet, I did not add a navigation test.

Are there tests you would recommend adding here? If so, do you have a code reference?

Thank you!
Ana